src/Components/Shifting/ListFilter.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from "../Common/HelperInputFields";
```

Comment:
Btw have you checked if these changes are to be done in Board View component in same directory? Now i noticed there is also board view

Reply:
The List Filter is common for both List View and Board View, It works!

Reply:
Ohk then great✅